stdlib/Random/src/misc.jl

```diff
@@ -306,28 +290,27 @@ optional `rng` argument specifies a random number generator (see
 
 # Examples
 ```jldoctest
-julia> randperm!(Xoshiro(123), Vector{Int}(undef, 4))
-4-element Vector{Int64}:
+julia> randperm!(Xoshiro(0), Vector{Int}(undef, 6))
+6-element Vector{Int64}:
+ 5
  1
- 4
  2
+ 6
  3
+ 4
 ```
 """
-function randperm!(r::AbstractRNG, a::Array{<:Integer})
+function randperm!(rng::AbstractRNG, a::Array{<:Integer})
```

Comment:
Any reason this, and `randcycle!` are not defined for `AbstractArray`?

Reply:
Interesting, I also just noticed that!